tests/functional/botocore/test_endpoints.py

```diff
@@ -13,68 +13,7 @@
 import pytest
```

Comment:
I think we are missing some of the functional tests from `tests/functional/configured_endpoint_urls/` directory in the original PR